web-client/src/views/CaseDetailEdit/Attachments.tsx

```diff
@@ -53,26 +53,33 @@ export const Attachments = connect(
                         icon={['fas', 'file-pdf']}
                         size="1x"
                       />
+                      {/* TODO: icon size doesn't match others */}
```

Comment:
will this be addressed in this PR or later?

Reply:
Its been addressed: [51bd0b6](https://github.com/ustaxcourt/ef-cms/pull/4622/commits/51bd0b66db0cf3d262fc7afd9cee21e89cef35ea)